src/domains/my/detail/components/PostList.jsx

```
@@ -13,10 +13,10 @@ const PostList = ({ mockData }) => {
     )
   }
 
-  const isAllChecked = mockData.length === checkedItems.length
+  const isAllChecked = boardData.length === checkedItems.length
```

ESLint

[error] 'boardData.length' is missing in props validation

(react/prop-types)